sdk/appcenter/src/main/java/com/microsoft/appcenter/utils/IdHelper.java

```diff
@@ -22,7 +22,7 @@
      * @return the installID
      */
     @NonNull
-    public static UUID getInstallId() {
+    public static synchronized UUID getInstallId() {
```

Comment:
No need for synchronized it as initialization is already guaranteed by the time distribute calls this. InstallId is called in finishConfiguration by core, which is before Distribute can get a reference to the channel. If no reference to channel then resume does nothing and is replayed by the applyEnabledState later.